models/articles-model.js

```diff
@@ -33,3 +33,27 @@ exports.addCommentByArticleId = (body, author, article_id) => {
     })
 }
 
+exports.updateArticlesVotes = (inc_votes, article_id) => {
```

Comment:
You can do this in one query without checking the number. If you add a positive number to an integer the number goes up. If you add a negative number to an integer the number goes down. So 2+3= 5, 5+ -2 = 3